README.md

```diff
@@ -102,9 +102,8 @@ See [the rustc-dev-guide for more info][sysllvm].
    list of options in `config.toml.example`.
 
    ```sh
-   printf 'profile = "user" \nchangelog-seen = 2 \n' > config.toml
+   ./configure --set changelog-seen=2 --set profile=user
```

Comment:
changelog-seen is already set to 2 by default when using the configure script (because it's not commented-out in config.toml.example).
```suggestion
   ./configure --set profile=user
```